spec/tSpec.js

```diff
@@ -64,6 +64,14 @@ describe('Ember.I18n.t', function() {
     expect(Ember.I18n.t('nothing.here')).to.equal('there.is.nothing.here.to.see');
   });
 
+  it('warns with a custom message that includes the passed context', function() {
+    Ember.I18n.missingMessage = function(key, context) {
```

Comment:
Since this is a global, you'll want to reset it to its default behavior after the spec.